maxim/logger/fireworks/client.py

```
@@ -0,0 +1,223 @@
+
+    def wrap_acreate(acreate_func):
+        """Wrapper for asynchronous chat completion create method.
+        
+        This wrapper function intercepts asynchronous chat completion requests
+        to Fireworks and adds comprehensive logging capabilities while
+        preserving the original API behavior.
+        """
+        @functools.wraps(acreate_func)
+        async def wrapper(self: ChatCompletion, *args: Any, **kwargs: Any):
+            # Extract Maxim-specific headers for trace and generation configuration
+            extra_headers = kwargs.get("extra_headers", None)
```

Ruff (0.12.2)

Use `kwargs.get("extra_headers")` instead of `kwargs.get("extra_headers", None)`

Replace `kwargs.get("extra_headers", None)` with `kwargs.get("extra_headers")`

(SIM910)